docs/configuration.md

```diff
@@ -1106,7 +1106,7 @@ Apart from these, the following properties are also available, and may be useful
     parameter allows an application to run multiple executors on the
     same worker, provided that there are enough cores on that
     worker. Otherwise, only one executor per application will run on
-    each worker.
+    each worker usually.
```

Comment:
```
    In standalone and Mesos coarse-grained modes, setting this
    parameter allows an application to launch multiple executors on the
    same worker in one schedule iteration, provided that there are enough
    cores on that worker. Otherwise, only one executor per application
    will be scheduled on each worker during one schedule iteration.
```